### PR TITLE
Added markdown for ToLower() string method

### LINKED ISSUE #1842: [Term Entry] C# Strings `.ToLower()`
### Reason/inspiration (optional)

We would like a new term entry for [C#](https://www.codecademy.com/resources/docs/c-sharp) about the `String.ToLower()` method.

The entry should go in a new file in docs/content/c-sharp/concepts/strings/terms/tolower/tolower.md

The entry should

- Have a syntax section illustrating the method with a ` ```pseudo ` code block.
- An example showing the method in use with a ` ```cs ` code block.
- A runnable codebyte example in a ` ```codebyte/csharp ` block.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by Rahul-Yadav-Online**
### Description

A C# entry has been created  for Strings.ToLower() method. This is the first "term" entry for C# Strings, so I have created a new folder labeled "terms" under c-sharp/concepts/strings. Closes #1842

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/c-sharp/concepts/strings/terms/tolower/tolower.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+ 
+
+## Syntax
+
+```pseudo
+string lowerCaseString = originalString.ToLower();
+```
+
+
+Here, `originalString` is the string you want to convert to lowercase, and `lowerCaseString` is the new string that contains the lowercase characters.
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/strings/terms/tolower/tolower.md:25]:
```suggestion
Here, `originalString` is the string being converted to lowercase, and `lowerCaseString` is the new string that contains the lowercase characters.
```

File: content/c-sharp/concepts/strings/terms/tolower/tolower.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+The **`ToLower()`** method is a string method in C# that returns a new string in which all the characters are converted to lowercase. The method does not modify the original string, but rather returns a new one. This method is helpful in normalizing user input or comparing strings in a case-insensitive manner.
+ 
+
+## Syntax
+
+```pseudo
+string lowerCaseString = originalString.ToLower();
+```
+
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/strings/terms/tolower/tolower.md:23]:
```suggestion
```
remove extra line break

File: content/c-sharp/concepts/strings/terms/tolower/tolower.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+Here, `originalString` is the string you want to convert to lowercase, and `lowerCaseString` is the new string that contains the lowercase characters.
+
+## Example
+
+In the following example, we create a new string `originalString` that contains the text *"HELLO WORLD"*. We then call the `ToLower()` method on this string and store the result in a new string variable `lowerCaseString`. Finally, we print the value of lowerCaseString to the console, which outputs *"hello world"*.
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/strings/terms/tolower/tolower.md:29]:
```suggestion
In the following example, a new string `originalString` is created that contains the text "HELLO WORLD". Then the `.ToLower()` method is called on this string and store the result in a new string variable `lowerCaseString`. Finally, the value of `lowerCaseString` is printed to the console, which outputs "hello world".
```

File: content/c-sharp/concepts/strings/terms/tolower/tolower.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+```cs
+string originalString = "HELLO WORLD";
+string lowerCaseString = originalString.ToLower();
+
+Console.WriteLine(lowerCaseString); // Output: "hello world"
+```
+
+## Codebyte Example
+
+In following runnable example, we create a new string `originalString` that contains the text *"THIS IS A TEST STRING"*. We then call the `ToLower()` method on this string and store the result in a new string variable `lowerCaseString`. Finally, we print both the original and the lowercase strings to the console.
```
Comment (SSwiniarski) [inline content/c-sharp/concepts/strings/terms/tolower/tolower.md:40]:
```suggestion
In following runnable example, a new string `originalString` is created that contains the text "THIS IS A TEST STRING". Then the `.ToLower()` method is called on this string and the result is stored in a new string variable `lowerCaseString`. Finally, both the original and the lowercase strings are printed to the console.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Rahul-Yadav-Online I did a first review and left some comments.

File: content/c-sharp/concepts/strings/terms/tolower/tolower.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **`ToLower()`** method is a string method in C# that returns a new string in which all the characters are converted to lowercase. The method does not modify the original string, but rather returns a new one. This method is helpful in normalizing user input or comparing strings in a case-insensitive manner.
+ 
+
```
[the remark this answers is not in the document]
Comment (Rahul-Yadav-Online) [reply content/c-sharp/concepts/strings/terms/tolower/tolower.md:17]: ok

File: content/c-sharp/concepts/strings/terms/tolower/tolower.md (hunk 2 above)
Comment (Rahul-Yadav-Online) [reply content/c-sharp/concepts/strings/terms/tolower/tolower.md:23]: Okay

File: content/c-sharp/concepts/strings/terms/tolower/tolower.md (hunk 1 above)
Comment (Rahul-Yadav-Online) [reply content/c-sharp/concepts/strings/terms/tolower/tolower.md:25]: do i need to remove the and as well ?

File: content/c-sharp/concepts/strings/terms/tolower/tolower.md (hunk 3 above)
Comment (Rahul-Yadav-Online) [reply content/c-sharp/concepts/strings/terms/tolower/tolower.md:29]: Okay will do

File: content/c-sharp/concepts/strings/terms/tolower/tolower.md (hunk 4 above)
Comment (Rahul-Yadav-Online) [reply content/c-sharp/concepts/strings/terms/tolower/tolower.md:40]: Got it, will change this one as well

File: content/c-sharp/concepts/strings/terms/tolower/tolower.md (hunk 1 above)
Comment (Rahul-Yadav-Online) [reply content/c-sharp/concepts/strings/terms/tolower/tolower.md:25]: okay

**Review by SSwiniarski — APPROVED** [review]
@Rahul-Yadav-Online Looks good for a second review.

**Review by yangc95 — APPROVED** [review]

**Comment by yangc95** [comment]
🙌 Congrats on this entry @Rahul-Yadav-Online here's the live link: https://www.codecademy.com/resources/docs/c-sharp/strings/tolower

Screenshot 2023-03-28 at 2 55 44 PM

> It looks as if you're anonymous. If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do.
>
> To add your GitHub username to your Codecademy profile:
>
> 1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:
>

>
> 2.) Then log out and log back into Codecademy using your GitHub login:
>

>
> If the steps above do not work, it may be due to an email or local configuration issue.
>
> In addition, we may reach out in the future if any adjustments to these entries are required.

cc: @SSwiniarski

**Pull request merged**